README.md

```diff
@@ -1,221 +1,276 @@
+[affects your bottom line](http://devnet.kentico.com/Blogs/Thomas-Robbins/September-2012/How-loading-time-affects-your-bottom-line-Infograp.aspx).
+*Asynchronous* ads do not block the page and can be delivered after core
+content - [Async FTW](http://www.krux.com/blog/krux-engineers/synchronous-versus-asynchronous-tags-whats-the-big-deal/).
 
+Why is it so hard to deliver ads asynchronously? Because they may contain calls to `document.write`, which expects to be handled synchronously.
+**PostScribe lets you deliver a synchronous ad asynchronously without modifying the ad code**.
 
 ### Approach
 
-Other tag writing libraries exist (see [alternatives](#alternatives)), but PostScribe is novel in it's use of what we call DOM Proxies, a way to ensure that the content is written as close to the way the browser would natively write the content with `document.write`/`innerHTML`. Read: it behaves just like the browser would, without convoluted parsing or hacks.
+Other tag writing libraries exist (see [alternatives](#alternatives)), but PostScribe is novel in its use of what we call DOM Proxies, a way to
+ensure that the content is written as close to the way the browser would natively write the content with `document.write`/`innerHTML`. Read: it
+behaves just like the browser would, without convoluted parsing or hacks.
 
 For more information:
 
-* [Presentation at HTML5devconf](http://youtu.be/ClzeilKwX10) by the author, Derek Brans
+* [Presentation at HTML5devconf](http://youtu.be/ClzeilKwX10) by the original author, Derek Brans
 * [Interactive Demo](http://run.plnkr.co/plunks/e8MJAx/) with side by side comparisons of other tag writers
-* [Documentation](https://github.com/krux/postscribe/tree/master/doc)
-* Browse the [raw](https://github.com/krux/postscribe/blob/master/postscribe.js) or [annotated](http://krux.github.com/postscribe/doc/postscribe.html) source code.
-
 
 # Getting Started
 
-PostScribe overrides document.write. It is best and safest to use PostScribe after DOM is ready.
+PostScribe overrides `document.write`. It is best and safest to use PostScribe after DOM is ready.
+
+## Installation
+
+### Browser
+
+If you just want to use the script without installing anything, use the following to load the script from *cdnjs*:
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/postscribe/2.0.0/postscribe.min.js"></script>
+```
+
+### NPM
+
+You can include `postscribe` using *npm*:
+
+```console
+npm install --save-dev postscribe
+```
+
+### Bower
+
+You can include `postscribe` using *bower*:
 
-Include `./htmlParser/htmlParser.js` and `./postscribe.js` on your page. TODO: cdn for postscribe.min.js
+```console
+bower install --save-dev postscribe
+```
+
+## Accessing
+
+### ES6/ES2015
+
+```javascript
+import {postscribe} from 'postscribe';
+```
+
+### AMD
+
+```javascript
+define(['postscribe'], function(postscribe) {
+
+});
+```
+
+### CommonJS
+
+```javascript
+var postscribe = require('postscribe');
```

Comment:
This echoes our thinking, and might be enough impetus to break the inertia :-)  If I do this, I'd like to change the signature of htmlParser to be a proper ES6 class.

On the disadvantaged:
1. We could fix the version if/when that becomes an issue.
2. I'm also not sure this is a primary use case.

Seems the way forward for modern JS development is using a toolchain like Webpack et al, so the advantages it provides for that is good and the disadvantage to oldschool users of both postscribe and htmlParser seems very limited.